api/src/telemetry/lib/get-report.test.ts

```diff
@@ -72,12 +86,12 @@ test('Runs and returns basic counts', async () => {
 	const report = await getReport();
 
 	expect(getItemCount).toHaveBeenCalledWith(mockDb, [
-		'directus_dashboards',
-		'directus_extensions',
-		'directus_files',
-		'directus_flows',
-		'directus_roles',
-		'directus_shares',
+		{ collection: 'directus_dashboards' },
+		{ collection: 'directus_extensions', where: ['enabled', '=', true] },
```

Comment:
Its not because of this PR but I noticed that we are actually counting bundles slightly wrong. Because a bundle is just a row and the inside-extensions point to it via its `id` a bundle with one extension inside counts as 2 and a bundle with two extensions inside counts as 3 - etc.

I looked up the original PR for the `get-item-count.ts` file, [here](https://github.com/directus/directus/pull/20825), which already introduced this behavior but I dont see it being discussed there. Did we decide that we are OK with this somewhere else? (cc @rijkvanzanten maybe you know?)

Now that we added a possibility for a where clause we might be able to filter those out! I personally would be in favor of doing so. :smile: :+1:

Reply:
Maybe theres an easier way but off the top of my head I came up with this:

```sql
SELECT COUNT(id)
FROM directus_extensions
WHERE id NOT IN (
  SELECT DISTINCT x1.id 
  FROM directus_extensions as x1
  INNER JOIN directus_extensions as x2 ON x1.id = x2.bundle
);
```

Reply:
A bundle should be counted as the sum of it's child extensions, excluding the parent 👍🏻

Reply:
Nice catch @DanielBiegler. 🔍 

Created a new util for extensions in https://github.com/directus/directus/pull/22337/commits/1ee6bf013a885722e35b281c12762398feeb2cab and patched it for partially enabled bundles in https://github.com/directus/directus/pull/22337/commits/bf12ce347cdc0fc6ea06da2ced203e05d40998e0.